### PR TITLE
Capstone 01:Africa AI summit 2023

### PR DESCRIPTION
**Pull request by zewdu444**
## Capstone 01:Africa AI summit 2023 

- [x] used correct GitHub flow.
- [x] used best practice guideline for html, CSS and JavaScript.
- [x]  Project setup and website deployed  [live link](https://zewdu444.github.io/conference_page/about.html).
- [x] Topic selected as "AFRICA AI SUMMIT 2023".
- [x] Content files like image  were imported into project folder including  Lato font.
- [x] Used mobile first approach and develop two pages (home and about).
- [x] Desktop version has been adapted for both pages.
- [x]  Flexbox and  Grid were used.
- [x] Mobile option were added.
- [x] Featured speaker data were loaded using JavaScript.
- [x] Readme file created as Microverse guide line.
- [x] Loom video were created to describe the project [live link].(https://www.loom.com/share/19d08633dd0343aa8738af9cd840c6d7).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,180 @@
...
+      </section>
+    </main>
+    <footer class="homefooter">
+      <img
+        class="imglogo"
+        src="./images/ailogo.png"
+        alt="afirca AI summit 2023 logo"
+      />
+      <p class="footerdetail">2023 Africa AI Conference.All rights reserved.</p>
+    </footer>
```
Comment (mckent05) [inline index.html:195]: - Good job done so far, but there are some little differences between your work and the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

Your Project:
- You have aligned the footer wrongly, place make sure the logo and text are aligned side by side 👍 
xtc

Design Template:
htewt

File: index.html [hunk 2]
```diff
@@ -8,7 +8,180 @@
...
+          </button>
+          <ul class="popnavlist">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#programs">Program</a></li>
+            <li><a href="#speakers">Speaker</a></li>
+            <li><a href="about.html#partners">Sponsor</a></li>
+            <li><a href="index.html">News</a></li>
+            <li><a href="index.html">AIevent</a></li>
+          </ul>
```
Comment (mckent05) [inline index.html:61]: **OPTIONAL** - I'd normally advise that it is not necessary to have different nav links for mobile and desktop versions, using CSS and media-queries you have have just one set of nav-links, style and hide all elements needed for both mobile and desktop version. Please fix this 👍 👍

File: index.html [hunk 3]
```diff
@@ -8,7 +8,180 @@
...
+        </div>
+      </nav>
+    </header>
+    <main class="maincontainer">
+      <section class="eventsection">
+        <!-- event description  -->
+        <h2 class="subtitle">Empowering Africa with AI</h2>
+        <h1 class="title">
+          AFRICAN CONFERENCE ON ARTIFICIAL INTELLIGENCE 2023
+        </h1>
```
Comment (mckent05) [inline index.html]: - For semantic reasons, it is important that your `hx` tags follow an ascending hierarchy. i.e, an `h2` tag should be used after an `h1` tag, not vice-versa. PLease fix this 🙏

File: index.html [hunk 4]
```diff
@@ -8,7 +8,180 @@
...
+            </button>
+            <button class="speakerbutton" id="lessbutton" hidden>
+              LESS <i class="fa-solid fa-angle-up"></i>
+            </button>
+          </div>
+        </div>
+      </section>
+      <!-- sponsor -->
+      <section class="sponsor homesponsor" id="partners">
+        <h2>Partner</h2>
```
Comment (mckent05) [inline index.html:160]: - Like I mentioned above, please ensure that all `hx` tags follow an ascending hierarchy 👍

File: index.html [hunk 5]
```diff
@@ -8,7 +8,180 @@
...
+            alt="addis ababa university"
+          />
+          <img
+            src="./images/EthiopianAI.png"
+            alt="Ethiopian Artificial Intelligence Institute"
+          />
+          <img src="./images/Presight_AI.png" alt="Presight AI" />
+          <img src="./images/Hawk university.png" alt="Hawk university" />
+          <img src="./images/IcogLab.png" alt="IcogLab" />
+        </div>
```
Comment (mckent05) [inline index.html]: - For semantic reasons, I suggest you use a `ul` tag to group your images and ensure all `img` tags are in a corresponding `li` tag. Please fix this 😄

File: about.html [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+            <i class="fa-2xl fa-solid fa-xmark"></i>
+          </button>
+          <ul class="popnavlist">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#programs">Program</a></li>
+            <li><a href="#speakers">Speaker</a></li>
+            <li><a href="#partners">Sponsor</a></li>
+            <li><a href="index.html">News</a></li>
+            <li><a href="index.html">AIevent</a></li>
```
Comment (mckent05) [inline about.html:59]: **OPTIONAL** - I'd normally advise that it is not necessary to have different nav links for mobile and desktop versions, using CSS and media-queries you have just one set of nav-links, style and hide all elements needed for both mobile and desktop versions. Please fix this 👍 👍. It'd also help to ensure you follow the DRY principle 😄

File: about.html [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="maincontainer">
+      <!-- event description  -->
+      <section>
+        <h2 class="subtitle aboutsubtitle">Empowering Africa with AI</h2>
+        <h1 class="title abouttitle">
```
Comment (mckent05) [inline about.html]: - For semantic reasons, it is important that your hx tags follow an ascending hierarchy. i.e, an h2 tag should be used after an h1 tag, not vice-versa. PLease fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @zewdu444, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your project looks good 👌, keep up the good work! 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (zewdu444) [reply index.html:195]: - [x] @mckent05  Many thanks for the feedback; the footer elements has been fixed.

File: index.html (hunk 3 above)
Comment (zewdu444) [reply index.html]: - [x] @mckent05 `hx` elements place in ascending order

File: index.html (hunk 5 above)
Comment (zewdu444) [reply index.html]: - [x] @mckent05 sponsor images has been warped in `ul` and `li`

**Review by ichala — APPROVED** [review]
Hi @zewdu444   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**